src/Deserializers/DataValueDeserializer.php

```diff
@@ -14,14 +14,14 @@
 /**
  * @since 0.1
  *
- * @license GPL-2.0+
+ * @license GPL-2.0-or-later
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Thiemo Kreuz
  */
 class DataValueDeserializer implements DispatchableDeserializer {
 
-	const TYPE_KEY = 'type';
-	const VALUE_KEY = 'value';
+	private const TYPE_KEY = 'type';
+	private const VALUE_KEY = 'value';
```

Comment:
Are you sure these are not used in some Wikibase code?

Reply:
Good point, [code search](https://codesearch.wmcloud.org/search/?q=DataValueDeserializer%3A%3A(TYPE%7CVALUE)_KEY) finds a handful of uses in [`SnakDeserializer`](https://github.com/wmde/WikibaseDataModelSerialization/blob/master/src/Deserializers/SnakDeserializer.php#L130).